specification/attestation/data-plane/Microsoft.Attestation/preview/2018-09-01-preview/attestation.json

```diff
@@ -258,17 +258,163 @@
                 }
             }
         },
+        "/operations/policy/certificates": {
+            "get": {
+                "tags": [
+                         "Policy Management Certificates"
+                        ],
+                "summary": "Retrieves the set of certificates used to express policy for the current tenant.",
+                "operationId": "PolicyCertificates_Get",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "An RFC7519 JSON Web Token structure containing an RFC7517 JsonWebKeySet which contains the certificates used to validate policy changes",
+                        "schema": {
+                            "type": "string"
+                        }
+                    },
+                    "400": {
+                        "description": "Bad request",
+                        "schema": {
+                            "$ref": "#/definitions/CloudError"
+                        }
+                    },
+                    "401": {
+                        "description": "Request is unauthorized",
+                        "schema": {
+                            "type": "string"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed",
+                        "schema": {
+                            "$ref": "#/definitions/CloudError"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                    "Get PolicyCertificates": {
+                      "$ref": "./examples/Get_PolicyCertificates.json"
+                    }
+                }
+            },
+            "put": {
+                "tags": [
+                    "Policy Management Certificates"
+                ],
+                "summary": "Adds a new attestation policy certificate to the set of policy management certificates.",
+                "operationId": "PolicyCertificates_Add",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "name": "policyCertificateToAdd",
+                        "in": "body",
+                        "description": "An RFC7519 JSON Web Token containing a claim named \"aas-policyCertificate\" whose value is an RFC7517 JSON Web Key which specifies a new key to add. The RFC7519 JWT must be signed with one of the existing signing certificates",
+                        "required": true,
+                        "schema": {
+                            "type": "string"
```

Comment:
Same applies here but for "consumes". application/json (with double-quotes) or text/plain?

Reply:
Yes, we expect (application/json with double quotes).